json/src/spec/authority_round.rs

```diff
@@ -46,14 +68,19 @@ pub struct AuthorityRoundParams {
 	/// a single block reward contract transition and is compatible with the multiple address option
 	/// `block_reward_contract_transitions` below.
 	pub block_reward_contract_transition: Option<Uint>,
-	/// Block reward contract address (setting the block reward contract overrides the static block
-	/// reward definition). This option allows to add a single block reward contract address and is
-	/// compatible with the multiple address option `block_reward_contract_transitions` below.
+	/// Block reward contract address which overrides the `block_reward` setting. This option allows
```

Comment:
~~This option allows to add~~
This provides the option to add

Reply:
I'm a bit mixed up. Can you clarify? The sentence loses its meaning after "and" if we rewrite it this way. Originally: this option allows and this option is compatible. With the change: this provides an option and this is compatible. In the latter case I'm not sure what "this" is referring to.

Reply:
Just attempting to make a complete sentence here rather than change the meaning- maybe would be better to say "this option allows for a single block reward contract..." or "this option allows the user to add" or another variation that completes the sentence and preserves the meaning?

Reply:
I think @andogro is saying that the wording "…allows to add…" is grammatically a bit off and that it'd improve the text if you reworded it with "…provides the option to add…" (or "set") here and on line 67 above. :)

Reply:
Would the issue be fixed by changing "...allows to add..." into "...allows one to add..."?

Reply:
Sure that works too.